mono/mini/driver.c

```diff
@@ -1976,7 +1975,7 @@ mono_main (int argc, char* argv[])
 	 */
 	mono_native_thread_set_name (mono_native_thread_id_get (), "Main");
 
-	if (enable_profile) {
+	if (mono_profiler_enabled ()) {
```

Comment:
Do we need to enable it at arg parsing time?

Can't we simply let mono_profiler_load set it? This reduces the amount of state we expose.